TaskBroker/VstsServerTaskBroker/IVstsReportingHelper.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace VstsServerTaskBroker
+{
+    public interface IVstsReportingHelper
```

Comment:
ReportingHelper does not look like a good here. We plan to do reporting work soon in RM